spec/requests/creator_authority_request_spec.rb

```diff
@@ -74,7 +76,7 @@
       get "/authorities/terms/creator_authority"
       expect(response).to have_http_status(:success)
       json_body = JSON.parse(response.body)
-      expect(json_body.count).to eq 13
+      expect(json_body.count).to eq 14
       expect(json_body.first["label"]).to eq "Cagetti, Marco"
```

Comment:
Unless the order is important here, we might want to change this to test for inclusion instead of matching on the first element:
```
expect(json_body).to include_json( label: "Cagetti, Marco")
```

see https://relishapp.com/waterlink/rspec-json-expectations/docs/json-expectations/include-json-matcher#expecting-json-string-to-include-simple-json

Reply:
Yes, order is important here (we expect them to be returned in alphabetical order).